docs/_posts/2024-09-02-exm-campaign-dispatch-stuck.md

```diff
@@ -0,0 +1,69 @@
+---
+title:  "EXM campaign dispatch stopped because CM instance crashed"
+date:   2024-08-29 08:00:00 +0100
+categories:
+- Sitecore
+tags:
+- Sitecore
+author: dgashi
+---
+
+![Email Marketing](../files/2024/09/02/email.jpg "Email Marketing")
+
+
+## Introduction
+
+
+In the world of digital marketing, email campaigns are a crucial tool for engaging with our audience, driving conversions, and building brand loyalty. 
+However, even the most meticulously planned campaigns can face unexpected challenges. Recently, we encountered a significant disruption when our system crashed mid-dispatch, causing our email campaigns to get stuck. Here’s a look at what happened, how we responded, and the lessons we learned from the experience.
+
+#### The Incident: When Things Went Off Track
+
+The campaigns were designed to reach thousands of subscribers with personalized content that would resonate with their interests.
+
+But then, the unexpected happened. Just as the second campaign of three was rolling out, our CM server crashed due to another issue. 
+
+The result? Hundreds of emails were stuck in limbo, neither reaching the inboxes nor bouncing back as undeliverable.
+
+
+#### Recovery: Getting Back on Track
+
+Once the system was back online, we had to decide how to proceed with the campaign. 
+
+Our options included resending the emails that were stuck or pausing the campaign altogether to avoid overwhelming our subscribers. After careful consideration, we chose to stagger the dispatch of the remaining emails. This allowed us to complete the campaign without bombarding recipients or risking further technical issues. We also took this opportunity to recheck the content and timing, ensuring that the messages were still relevant and effective.
+
+#### Solution
+
+Since the campaign with the status "Sending" stopped with a progress of 40%, we wanted to be sure and reach out to the Sitecore Support team to find the best technical solution. 
+
+![EXM Master](../files/2024/09/02/exm-campaign-status.png "Database snippet")
+
+In this cases we need to manually change the states. Pausing and Resuming of the campaings was not possible in our case. Clicking the Button in the Delivery tab of the campaings did not change anything.
+
+In this case, we need to manually change the states. Pausing and resuming the campaign was not possible in our case. Clicking the Button in the Delivery tab of the campaign did not change anything.
+
+So manually modifying the related message status in the EXM.Master DB helps.
+
+Here is the SQL script we used:
+
+~~~~sql
+USE [Sitecore.Exm.master]
+GO
+
+UPDATE [dbo].[Campaigns]
+SET [Status] = 'Paused'
+WHERE MessageID = 'Your campaign id'
+GO
+~~~~
```

Comment:
Hi @drendotnet,

I suggest using the liquid tag for highlighting, it doesn't always work with other highlights.
This could be changed as follows:
```suggestion
{% highlight sql %}
USE [Sitecore.Exm.master]
GO

UPDATE [dbo].[Campaigns]
SET [Status] = 'Paused'
WHERE MessageID = 'Your campaign id'
GO
{% endhighlight %}
```